agentinstall.ps1

```diff
@@ -64,14 +64,25 @@ wget "https://vstsagentpackage.azureedge.net/agent/$agentVersion/$zip" -OutFile
 
 
 for ($i = 1; $i -le $count; $i++) {
+    Set-Location C:\a
     $agentDir = "A$i"
-    $agentName = "$ComputerName-A$i"
+    $agentName = "$ComputerName-ws2019-A$i"
     
     Expand-Archive -Path ./$zip -DestinationPath ./$agentDir
     
     cd $agentDir
         
-    .\config.cmd --unattended --url https://dev.azure.com/$account --auth PAT --token $PAT --pool "$pool" --agent $agentName --runAsService
-    
+    .\config.cmd --unattended --url https://dev.azure.com/$account --auth PAT --token $PAT --pool "$pool Agents" --agent "$agentName" --runAsService
```

Comment:
I don't think pool is the correct variable name here, I read it and thought it was the full pool name. $PoolNamePrefix?

Reply:
done